lib/tokenize.cpp

```diff
@@ -1928,7 +1928,7 @@ void Tokenizer::combineStringAndCharLiterals()
 {
     // Combine wide strings and wide characters
     for (Token *tok = list.front(); tok; tok = tok->next()) {
-        if (Token::Match(tok, "[Lu] %char%|%str%")) {
+        if (Token::Match(tok, "L|u|U|u8 %char%|%str%")) {
```

Comment:
`Token::Match()` does not handle `[LuU]|u8` or `u8|[LuU]`.

Reply:
Also, I was unsure what `isLong()` meant for a string. Larger than `char`? I.e., I don't think the `u8` prefix should be long then?

Reply:
You can't write `[...]|u8` currently. I would not be against that this feature is added, of course.

Yes I think the `isLong()` should be avoided. A type of a string literal that has `isLong` is `wchar_t *` if I remember correctly.